### DISCUSSION TITLE
Examples for adding bespoke plugins

### DISCUSSION THREAD

**Discussion by mil-ad** [marp-team/marp#371] [Q&A] [answered]
It'd be great if we could include examples in the docs for how to add additional bespoke plugins (if that's even possible). I'd love to try some but no idea how to integrate it with marp

**Comment by yhatt** [comment, marked as answer]
Unfortunately adding bespoke plugin is not allowed to CLI users. The `bespoke` template provided by Marp CLI is made up of [Bespoke.js](http://markdalgleish.com/projects/bespoke.js/) framework and [tailor-made bespoke plugins designed for Marp](https://github.com/marp-team/marp-cli/blob/7e58d83b12d4b8dc0c738e539378e3993bbe85f0/src/templates/bespoke/bespoke.ts#L39-L53), and not considered about compatibllity with [general bespoke plugins](https://www.npmjs.com/search?q=keywords%3Abespoke-plugin).

If still have interested, you can [build Marp CLI from the source](https://github.com/marp-team/marp-cli/blob/main/.github/CONTRIBUTING.md) that has added plugins your brought to `bespoke.ts`.
